tests/containers/integration-test-base

```diff
@@ -0,0 +1,13 @@
+FROM quay.io/centos/centos:stream9
+
+RUN dnf install -y dnf-plugin-config-manager
+
+RUN dnf install \
+        systemd-devel \
+        systemd \
+        selinux-policy \
+    -y
+
+RUN dnf -y clean all
+
+WORKDIR /hirte
```

Comment:
Do we need this directory for anything? Isn't it only remain from the time before conversion to tmt powered tests?